tdrs-backend/tdpservice/fixtures/cypress/users.json

```diff
@@ -90,5 +90,156 @@
             "user": "78e10c5b-271e-4838-9e7b-b3c30773ef2e",
             "region": 5
         }
+    },
+    {
+        "model": "users.user",
+        "pk": "7b7c4778-17e4-49a1-83b8-0e6a962972e7",
+        "fields": {
+          "password": "",
+          "last_login": null,
+          "is_superuser": false,
+          "username": "fra-data-analyst-fred@teamraft.com",
```

Comment:
could we prepend `cypress-` to the emails to help distinguish the users? `cypress-fra-data-analyst-fred@teamraft.com`